### PR TITLE
Add ipfs-deploy

### PR DESCRIPTION
**Pull request by agentofuser**
https://github.com/agentofuser/ipfs-deploy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ Sniffs the DHT gossip and indexes file and directory hashes. Metadata and search
 
 ## Tools
 
+- [@agentofuser/ipfs-deploy](https://github.com/agentofuser/ipfs-deploy) - Zero-config CLI to deploy static websites: cd my-static-website && npx @agentofuser/ipfs-deploy
```
Comment (koalalorenzo) [inline README.md]: Should we call it `Ipfs-deploy` ?

Comment (agentofuser) [reply README.md]: @koalalorenzo I changed it to `ipfs-deploy` only. Thank you!

**Comment by autonome** [comment]
@koalalorenzo Is this good to merge?

**Comment by koalalorenzo** [comment]
@autonome I think so, what do you think? :)

**Comment by autonome** [comment]
@koalalorenzo The ccontent looks good to me, but this is my first run through the format and structure of this site, so I'll leave that to your expertise :D

**Pull request merged**